site/src/sections/Navbar/navbar.style.js

```diff
@@ -0,0 +1,67 @@
+import styled from "styled-components";
+
+export  const NavStyleWrapper = styled.nav`
+    
+    .nav{
+        position: sticky;
+    }
```

Comment:
I don't think this class is used anywhere

Reply:
removed it :+1: